zerver/management/commands/restore_messages.py

```diff
@@ -22,17 +22,23 @@ class Command(ZulipBaseCommand):
 
 Examples:
 To restore all recently deleted messages:
-  ./manage.py restore_messages
+  ./manage.py restore_messages --restore-all --restore
```

Comment:
I don't think the `--restore` is right, here?  I don't see that argument.
```suggestion
  ./manage.py restore_messages --restore-all
```

Reply:
Yeah, meant to have `--restore-deleted` as the 2nd arg, thanks for catching